### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arshidkv12**

### CODE REVIEW

**Comment by jiacai2050** [comment]
@arshidkv12 The CI failed, please execute `make lint`.

**Comment by jiacai2050** [comment]
@arshidkv12 Hi, It seems this project is just hello world demo for write PHP extension in Zig, not a web framework, maybe you can put it under [FFI](https://github.com/zigcc/awesome-zig#ffi-bindings)

**Comment by arshidkv12** [comment]
Thank you

File: README.md [hunk 1]
```diff
@@ -313,6 +313,7 @@ TOC is generated by [markdown-toc](https://github.com/jonschlinkert/markdown-toc
 - [jiacai2050/zig-curl](https://github.com/jiacai2050/zig-curl) : Zig bindings for libcurl
 - [jiacai2050/zig-rocksdb](https://github.com/jiacai2050/zig-rocksdb) : Zig bindings for RocksDB.
 - [jiacai2050/zig-jemalloc](https://github.com/jiacai2050/zig-jemalloc) : Zig allocator baked by jemalloc
+- [PHP extension written in Zig](https://github.com/arshidkv12/zig-php)
```
Comment (jiacai2050) [inline README.md:316]:
```suggestion
- [arshidkv12/zig-php](https://github.com/arshidkv12/zig-php) : Write PHP extension in Zig
```

**Review by jiacai2050 — APPROVED** [review]

**Pull request merged**